### PR TITLE
PLANET-6940 Convert Deep Dive pattern into a block template

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-6940](https://jira.greenpeace.org/browse/PLANET-6940)

### Testing

You should be able to add a Deep Dive pattern as before without any problems! And the layouts using it should also still work as expected.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [oberon](https://www-dev.greenpeace.org/test-oberon) | [admin](https://www-dev.greenpeace.org/test-oberon/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-oberon/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-oberon) | [composer-local.json](https://github.com/greenpeace/planet4-test-oberon/blob/main/composer-local.json)

:watch: 2022.11.09 13:08:47

File: assets/src/block-templates/quick-links/template.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const category = ['core/column', {}, [
       level: 5,
       style: {typography: {fontSize: '1rem'}},
       textAlign: 'center',
-      placeholder: 'Category'
+      placeholder: __('Category', 'planet4-blocks-backend')
```
Comment (mleray) [inline assets/src/block-templates/quick-links/template.js:16]: I forgot to add a translation here before merging 😬

File: assets/src/block-templates/quick-links/template.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const template = ({
   backgroundColor = 'grey-05',
 }) => ([
   ['core/group', {
-    className: `block ${metadata.classname}`,
+    className: 'block',
```
Comment (mleray) [inline assets/src/block-templates/quick-links/template.js:26]: This `metadata.classname` resulted in `undefined`, and it seems that we don't need it

**Review by sagarsdeshmukh — APPROVED** [review]
Tested on local and test instance 🥇 
Looks good to me 👍

**Pull request merged**